src/components/GestureButtons.tsx

```diff
@@ -1,65 +1,108 @@
-/* eslint-disable eslint-comments/no-unlimited-disable */
-/* eslint-disable */
-// @ts-nocheck TODO(TS) provide types
-import PropTypes from 'prop-types';
 import React from 'react';
-import { Animated, Platform, processColor, StyleSheet } from 'react-native';
+import {
+  Animated,
+  Platform,
+  processColor,
+  StyleSheet,
+  StyleProp,
+  ViewStyle,
+} from 'react-native';
 
 import createNativeWrapper from '../handlers/createNativeWrapper';
 import GestureHandlerButton from './GestureHandlerButton';
 import State from '../State';
 
-export const RawButton = createNativeWrapper(GestureHandlerButton, {
-  shouldCancelWhenOutside: false,
-  shouldActivateOnStart: false,
-});
+import {
+  GestureEvent,
+  HandlerStateChangeEvent,
+} from '../handlers/gestureHandlers';
+import {
+  NativeViewGestureHandlerPayload,
+  NativeViewGestureHandlerProperties,
+} from '../handlers/NativeViewGestureHandler';
+
+/* BUTTONS PROPERTIES */
+
+interface RawButtonProperties extends NativeViewGestureHandlerProperties {
+  exclusive?: boolean;
+  testID?: string;
+  accessibilityLabel?: string;
```

Comment:
There are more a11y props to consider, see `AccessibilityProps` interface in RN types.

Reply:
It is taken from the former `rngh.d.ts` file. I believe it is not needed there since it is used in `GenericTouchable.tsx`, where other a11y props are provided and used.

Reply:
But Touchables aren't buttons, right? So we should somehow support every other a11y prop in Buttons.